src/plopp/graphics/interactive.py

```diff
@@ -75,6 +75,9 @@ def __getattr__(self, key):
         except AttributeError:
             return getattr(self._fig, key)
 
+    def __dir__(self):
+        return list(set(dir(VBox) + dir(super()) + dir(self._fig)))
```

Comment:
Technically also `self.left_bar`, etc. Are those actually part of the public API?

Reply:
Those are listed by the `dir(super())`, strangely enough...

Reply:
I added a `sorted` now.